src/AutoRest.CodeModel/Program.cs

```diff
@@ -19,10 +19,10 @@ internal static class Program
         private static void Main()
         {
             using var webClient = new WebClient();
-            webClient.DownloadFile(@"https://raw.githubusercontent.com/Azure/perks/master/codemodel/.resources/all-in-one/json/code-model.json", "../../../../../src/AutoRest.CodeModel/code-model.json");
+            var schemaJson = webClient.DownloadString(@"https://raw.githubusercontent.com/Azure/perks/master/codemodel/.resources/all-in-one/json/code-model.json");
 
-            var schemaJsonLines = File.ReadAllLines("../../../../../src/AutoRest.CodeModel/code-model.json");
-            var schemaJson = String.Join(Environment.NewLine, schemaJsonLines)
```

Comment:
The code-model schema was being cached into the repo so that we could keep track of the file we generated the CodeModel.cs from. That file has no versioning or anything, so this was just a simple mechanism to track that and see changes to the schema when they happen.

Reply:
Fixed